docs/epix100_geometry.ipynb

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
<p>I'd normally expect the first pixel to be on one of the outside corners, not in the centre. Unless the data is stored in a really weird way, I think the ASICs are in the wrong orientation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/European-XFEL/EXtra-geom/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Good question. I slice them in the order they labeled in docs. But I don't flip them as it did the controller when assembling data in the single array.

I just didn't think that anyone really needs ASICs flipped to match the electronic layout. Do you think it is necessary?

Reply:
How is the data stored in the files? The 'first pixel' should correspond to the data you access with `frame[0, 0]` (and the first row is from there along `frame[0, 1]`, `frame[0, 2]` etc.).

Reply:
There are certain conventionality in how to slices tiles. You can do this to match the labeling accepted in the documentation, or align the ASIC electrical circuits, or as you suggest be led by packing tiles in data.

I have fixed the code to match the last one. And added support of ePix10k